cpp/src/Ice/ConnectionI.h

```diff
@@ -268,6 +258,11 @@ namespace Ice
         void initiateShutdown();
         void sendHeartbeatNow();
 
+        void sendResponse(OutgoingResponse, std::uint8_t compress);
+        void sendResponse(std::int32_t, Ice::OutputStream*, std::uint8_t);
```

Comment:
These 3 functions used to be the ResponseHandler. They are called by the first sendResponse; we could refactor further and merge sendResponse + sendNoResponse into the main sendResponse.